docs/reference/announcements.md

```diff
@@ -8,9 +8,7 @@ description: "Important announcements including deprecation & removal notices"
 
 ### Syntax changes in Helm chart
 
-:::danger
-The Camunda Helm chart upgrade to v10.0.0 and 10.0.1 will cause gRPC endpoints to disconnect. Do not upgrade and wait until further notice.
-:::
+The Camunda Helm chart upgrade to v10.0.0 and 10.0.1 will cause gRPC endpoints to disconnect. It's not possible to upgrade from v9.x.x to v10.0.0 or v10.0.1, instead upgrade directly to v10.0.2 or above.
```

Comment:
I would not emphasise on an issue with releases no longer available. 
The Camunda Helm charts v10.0.0 and 10.0.1 have been removed to avoid breaking issues. Camunda Helm Chart v10.0.2 is a stable release.

Reply:
Cool - Updated the wording!